src/App.jsx

```diff
@@ -18,6 +19,7 @@ function App() {
         <Route path="/resources" component={""} />
         <Route path="/contact" component={""} />
         <Route path="/login" component={""} />
+        <Route path="/" component={Footer} />
```

Comment:
is this correct that your home page routes to footer? @tmansuroglu

Reply:
No it is not. We have home page container component for `path="/`

Reply:
Hi, it not exact so it will be rendered on every page.